openstack/designate/templates/api-deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             - name: DEPENDENCY_JOBS
               value: "designate-migration"
             - name: DEPENDENCY_SERVICE
-              value: "designate-mariadb,{{ .Release.Name }}-rabbitmq"
+              value: "designate-mariadb,rabbitmq"
```

Comment:
why this change? I did not introduce that dependency in Nova, Neutron, Cinder because the API should not be blocked by audit-specific components. If the notifications-rabbit is not up in time, the middleware will buffer.

Reply:
It already had the requirement, I renamed it when I was renaming the rabbit. Removed the change.